packages/byods/src/byods/index.ts

```diff
@@ -37,9 +37,14 @@ export default class BYODS {
    * @example
    * const sdk = new BYODS({ clientId: 'your-client-id', clientSecret: 'your-client-secret' });
    */
-  constructor({clientId, clientSecret}: SDKConfig) {
-    this.config = {clientId, clientSecret};
-    this.tokenManager = new TokenManager(clientId, clientSecret);
+  constructor({clientId, clientSecret, tokenStorageAdapter}: SDKConfig) {
+    this.config = {clientId, clientSecret, tokenStorageAdapter};
+    this.tokenManager = new TokenManager(
+      clientId,
+      clientSecret,
+      PRODUCTION_BASE_URL,
```

Comment:
Over here, I had to add this dummy value in order to pass the `tokenStorageAdapter` as well. I also noticed that we are not passing any base url into the token manager.
Should we be creating this after setting up our base urls? Thereby passing it into the manager? Just one question.

Reply:
Move it after env parsing